memif-new/_examples/responder/go.mod

```diff
@@ -0,0 +1,10 @@
+module github.com/responder
+
+go 1.16
+
+require (
+	github.com/pkg/profile v1.6.0
+	memif v0.0.0
+)
+
+replace memif v0.0.0 => /home/rtan/Work_dir/FD.io/memif/memif-new/memif
```

Comment:
I would recommend that you use relative directories here rather than absolute:

../